docs/ref/templates/builtins.txt

```diff
@@ -2430,18 +2430,15 @@ slightly different call:
     {% static "images/hi.jpg" as myphoto %}
     <img src="{{ myphoto }}"></img>
 
-.. note::
+.. versionchanged:: 1.10
```

Comment:
Keep in mind that these notes are removed in 2 two release, so they should be "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features